src/assets/icons/index.tsx

```diff
@@ -0,0 +1,27 @@
+import {ReactComponent as Profile} from './profilePic.svg';
+import {ReactComponent as DeleteIcon} from './delete.svg';
+import {ReactComponent as EditIcon} from './edit.svg';
+import {ReactComponent as MailIcon} from './mail.svg';
+import {ReactComponent as PasswordIcon} from './password.svg';
+import {ReactComponent as NameIcon} from './name.svg';
+import {ReactComponent as LastNameIcon} from './lastName.svg';
+import {ReactComponent as ConfirmPasswordIcon} from './confirmPassword.svg';
+import {ReactComponent as LogoutIcon} from './logout.svg';
+import {ReactComponent as ErrorIcon} from './error.svg';
+import {ReactComponent as SearchIcon} from './search.svg';
+
+
+export const LOGO = '/icons/logo.jpeg';
```

Comment:
keep the logo jpeg image in a different folder. Let's keep the icons folder exclusively for svg icons